### ISSUE TITLE
5.0.7 - Schedules configured to use Computer Group do not run

### ISSUE THREAD

**Issue by bt-ndollimount** [ironmansoftware/powershell-universal#3768]
### Version

5.0.6

### Severity

Medium

### Environment

msi

### Steps to Reproduce

When a scheduled job is configured to run on a computer group, the schedule does not run, neither on the schedule or when triggering the schedule.

### Expected behavior

```console
The job should run on the computer group.
```

### Actual behavior

```console
The job does not run; no job is created.
```

### Additional Environment data

Version 5.0.7

**Comment by bt-ndollimount** [comment]
@adamdriscoll This appears to still be an issue in 5.0.11. I noticed that a couple of our scripts hadn't run since September 25. When I checked them, they're the only two I had set to use a Computer Group. Manually triggering the schedule isn't working either.

**Comment by adamdriscoll** [comment]
I'm curious if this is a collation problem with PGSQL where case sensitivity may be a problem or something. Either way, I will re-investigate.

**Comment by bt-ndollimount** [comment]
@adamdriscoll Appreciate it. We're in the process of rolling out an externally accessible node solely for running the dashboard and apps for our users and we don't want it to be processing job. So this functionality will be pretty important for us to be working in the near future🫡.

**Comment by adamdriscoll** [comment]
Found the root cause. This will be fixed in 5.0.12.
